tests/integ/test_udf.py

```diff
@@ -1496,10 +1542,6 @@ def test_add_packages_negative(session, caplog):
         session.remove_package("python-dateutil")
     assert "is not in the package list" in str(ex_info)
 
-    with pytest.raises(ValueError) as ex_info:
```

Comment:
Unavailable packages no longer have a straightforward error; this case is covered in other tests.